lib/r10k/module/git.rb

```diff
@@ -147,7 +146,8 @@ def validate_ref(desired, default, default_override)
         msg << "or resolve default ref '%{default}'"
         vars[:default] = default
       else
-        msg << "and no default provided"
+        msg << "and no default provided. r10k no longer hardcodes 'master' as the default ref. "
+        msg << "Consider setting git:default_ref in your r10k config."
```

Comment:
I think we should suggest setting `ref` on a per-module basis, rather than suggesting the global `default_ref` setting. Or at least give both options.